autopush/web/validation.py

```diff
@@ -320,8 +321,8 @@ def validate_auth(self, d):
 
         try:
             jwt = extract_jwt(token, public_key)
-        except ValueError:
-            raise InvalidRequest("Invalid Authorization Header",
+        except (ValueError, JOSEError) as ex:
+            raise InvalidRequest("Invalid Authorization Header: %s" % str(ex),
```

Comment:
I was kinda debating that. In interests of clarity and developer relationship, it wouldn't hurt. From a more paranoid PoV, it's never a good idea to give more data than absolutely necessary. I'm trying to be less paranoid, but am absolutely fine with not disclosing the actual error.